source/teleop.rst

```diff
@@ -82,5 +89,6 @@ package.
 
 ::
 
-	>$ rosrun teleop_twist_keyboard teleop_twist_keyboard.py
+  >$ export ROS_MASTER_URI=http://<robot_name_or_ip>:11311
```

Comment:
these appear to be 2 spaces -- if I recall, they need to be at least 3 for the sphinx to render them as indented properly